evm/utils/blocks.py

```diff
@@ -26,12 +32,27 @@ def persist_block_to_db(db, block):
         rlp.encode(block.hash, sedes=rlp.sedes.binary),
     )
 
+
+def get_score(db, block_hash):
+    if block_hash == GENESIS_PARENT_HASH:
+        return GENESIS_DIFFICULTY
```

Comment:
This seems like a potential "future" problem as all chains will not start with the same genesis difficulty.  Is this something that can be EVM aware so that it can pull the genesis block and pull the genesis difficulty from there?